.github/workflows/integration.yml

```diff
@@ -180,18 +196,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
-      - name: Use Node.js 15.x
-        uses: actions/setup-node@v2
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v3
         with:
-          node-version: 15.x
+          node-version: 16.x
 
       - name: Message commit
         run: echo "is RELEASE => ${{ github.event.head_commit.message }} !!"
 
       - name: Get release version
-        run: echo ::set-output name=VERSION::$(node -e "console.log(require('./package.json').version);")
+        # run: echo ::set-output name=VERSION::$(node -e "console.log(require('./package.json').version);")
```

Comment:
Remove commented out?

Reply:
Yep my bad, I forgot it

Reply:
I just found out there is a problem with this change : the tag created on [this line](https://github.com/iTowns/itowns/blob/7f86d26e184d3b64981d6c8767c195e8c2ab6a79/.github/workflows/integration.yml#L205) lacks the version number. It create a `v` tag instead of `v2.xx.xx`. I'll add that to #1998, so that this PR can be merged to have a working CI.